app/controllers/categories_controller.rb

```diff
@@ -1,10 +1,21 @@
 class CategoriesController < ApplicationController
   def index
     render locals: {
-      categories: Category.all
+      categories: Category.all.where(parent_category_id: nil)
     }
   end
-  
+
+  def show
+    category = Category.find_by(parent_category_id: params.fetch(:id))
```

Comment:
[:id] not (:id) - not sure how much this matters but that's how we've always done it